Android/app/src/main/java/app/intra/socks/OverrideSocksHandler.java

```diff
@@ -164,11 +166,17 @@ public void doConnect(Session session, CommandMessage commandMessage) throws IOE
       // We had a functional socket long enough to record statistics.
       // Report the BYTES event :
       // - VALUE : total transfer over the lifetime of a socket
```

Comment:
Could you clarify what "transfer" is? It seems that's downloaded + uploaded bytes?

Reply:
Done